### PR TITLE
Add Öffi to Native Apps (open source)

### PR DESCRIPTION
**Pull request by schildbach**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,6 +143,7 @@ Apps people use when taking transit.
 - [OpenTripPlanner Android](https://github.com/CUTR-at-USF/OpenTripPlanner-for-Android/wiki) - An Android app for [OpenTripPlanner](http://www.opentripplanner.org/)
 - [OpenTripPlanner iOS](https://github.com/opentripplanner/OpenTripPlanner-iOS) - An iOS app for [OpenTripPlanner](http://www.opentripplanner.org/)
 - [Transportr](https://github.com/grote/Transportr) An Android app that uses [public-transport-enabler](https://github.com/schildbach/public-transport-enabler) in order to connect to many different transport networks worldwide.
+- [Öffi](https://gitlab.com/oeffi/oeffi) (Android) - The King of Public Transit!
```
Comment (barbeau) [inline README.md:146]: Could you please change this to a more descriptive and objective summary of the app?  Where it's deployed, what it does, the language it's written in, etc. would be useful.

**Review by barbeau — CHANGES_REQUESTED** [review]
@schildbach Thanks for submitting this! A requested change inline.  Please also sign the CLA, thanks!

File: README.md (hunk 1 above)
Comment (schildbach) [reply README.md:146]: What about this?

"Öffi tells you where and when trains and busses go, including delays and replacement bus service for more and more transport authorities in Europe and beyond. It's written in Java and installed on 10M+ devices."

I won't sign CLAs though, so probably someone else needs to PR this.

**Comment by barbeau** [comment]
@schildbach Thanks, I've added it myself in https://github.com/CUTR-at-USF/awesome-transit/pull/137.

**Pull request closed without merging**